make/development-environments.mk

```diff
@@ -15,6 +15,10 @@ install-metallb: kustomize yq ## Installs the metallb load balancer allowing use
 	kubectl -n metallb-system wait --for=condition=ready pod --selector=app=metallb --timeout=60s
 	./utils/docker-network-ipaddresspool.sh kind $(YQ) ${SUBNET_OFFSET} | kubectl apply -n metallb-system -f -
 
+.PHONY: install-observability-crds
+install-observability-crds: $(KUSTOMIZE)
+	$(KUSTOMIZE) build ./config/observability/| $(CONTAINER_ENGINE) run --rm -i docker.io/ryane/kfilt -i kind=CustomResourceDefinition | kubectl apply --server-side -f -
```

Comment:
RH does not meet `docker.io` license agreements. Can we still use it within kuadrant? 

If not, we can find an alternative way to do this, either using `yq, `jq` or something else (maybe kustomize??)